docs/examples/swift/iOS-Carthage/Cartfile.resolved

```diff
@@ -1,2 +1,2 @@
-github "cossacklabs/themis" "0.13.0"
+github "cossacklabs/themis" "gothemis/v0.13.0"
```

Comment:
this is truly funny

Reply:
> ```diff
> -github "cossacklabs/themis" "0.13.0"
> +github "cossacklabs/themis" "gothemis/v0.13.0"
> ```

No idea why this happens. It does not happen with `"0.12.0"`, for example, which also has `gothemis/v0.12.0` pointing to the same commit. Maybe Carthage does not like annotated signed tags?..

Well, I guess we can leave it like this since that's what users will be get in their projects. And the tags aren't going anywhere. But I'd like to have some fairies come and fix it ☁️